planning/mission_planner/src/lanelet2_plugins/utility_functions.cpp

```diff
@@ -14,15 +14,44 @@
 
 #include "utility_functions.hpp"
 
+#include <boost/geometry.hpp>
+
 #include <lanelet2_core/geometry/Lanelet.h>
 
 #include <unordered_set>
+#include <utility>
 
 bool exists(const std::unordered_set<lanelet::Id> & set, const lanelet::Id & id)
 {
   return set.find(id) != set.end();
 }
+tier4_autoware_utils::LinearRing2d create_vehicle_footprint(
+  const vehicle_info_util::VehicleInfo & vehicle_info)
+{
+  using tier4_autoware_utils::LinearRing2d;
+  using tier4_autoware_utils::Point2d;
+
+  const auto & i = vehicle_info;
 
+  LinearRing2d footprint;
+  footprint.push_back(Point2d{i.max_longitudinal_offset_m, i.max_lateral_offset_m});
+  footprint.push_back(Point2d{i.max_longitudinal_offset_m, i.min_lateral_offset_m});
+  footprint.push_back(Point2d{i.min_longitudinal_offset_m, i.min_lateral_offset_m});
+  footprint.push_back(Point2d{i.min_longitudinal_offset_m, i.max_lateral_offset_m});
+
+  return footprint;
+}
```

Comment:
FYI: I will add this function to vehicle_info_util. With another PR, you can replace this function.
https://github.com/autowarefoundation/autoware.universe/pull/2394

Reply:
Thanks, i will update it :+1: